airbyte-integrations/connectors/source-airtable/acceptance-test-config.yml

```diff
@@ -1,20 +1,28 @@
 # See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
 # for more information about how to configure these tests
 connector_image: airbyte/source-airtable:dev
-tests:
+acceptance_tests:
   spec:
-    - spec_path: "source_airtable/spec.json"
+    tests:
+      - spec_path: "source_airtable/spec.json"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
+    tests:
+      - config_path: "secrets/config.json"
+      # bypassed this check, because discovery mechanism was changed
+        backward_compatibility_tests_config:
```

Comment:
Could you please share the differences between the previous catalog structure and the new one?
Can we find a way to make the new catalog backward compatible? This change might require all our users to reset their data and refresh their schema.

Reply:
The biggest difference is what the schema will contain in the end. Previously it was built using a single complete row of data, omitting empty columns and reflecting wrong data structure
Now we fetch the correct schema and keep the empty columns in place, providing correct data structure.